vllm/entrypoints/openai/api_server.py

```diff
@@ -228,4 +346,4 @@ async def authentication(request: Request, call_next):
                 ssl_keyfile=args.ssl_keyfile,
                 ssl_certfile=args.ssl_certfile,
                 ssl_ca_certs=args.ssl_ca_certs,
-                ssl_cert_reqs=args.ssl_cert_reqs)
+                ssl_cert_reqs=args.ssl_cert_reqs)
```

Comment:
This line as well, please avoid it in the future

Reply:
> This line as well, please avoid it in the future
Sure. I will keep in mind.